testutil/ciao-down/workload.go

```diff
@@ -76,6 +81,40 @@ func (wkld *workload) save(ws *workspace) error {
 }
 
 func loadWorkloadData(ws *workspace, workloadName string) ([]byte, error) {
+	var absPath string
+	u, err := url.Parse(workloadName)
+	if err == nil {
+		switch u.Scheme {
+		case "http", "https":
+			workloadFile, err := ioutil.TempFile("/tmp", ".workload")
+			if err != nil {
+				break
+			}
+
+			absPath = workloadFile.Name()
+			defer os.Remove(absPath)
+
+			// 10 seconds should be enough to download the workload file
+			ctx, cancelFunc := context.WithTimeout(context.Background(), 10*time.Second)
+			defer cancelFunc()
+			err = getFile(ctx, workloadName, workloadFile, downloadProgress)
+			if err != nil {
+				return nil, fmt.Errorf("Unable download workload file from %s: %v", workloadName, err)
+			}
+		case "file":
+			absPath = path.Join("/", u.Host, u.Path)
+		default:
+			absPath, _ = filepath.Abs(u.Path)
+		}
+	}
+
+	if strings.Compare(absPath, "") != 0 {
+		wkld, err := ioutil.ReadFile(absPath)
+		if err == nil {
+			return wkld, nil
```

Comment:
Isn't that bound to fail though?  

The variable localPath would get initialised to something weird

```
localPath := filepath.Join(ws.Home, ".ciao-down", "workloads",
		fmt.Sprintf("%s.yaml", workloadName))
```

localPath would end up being "http://myurl.com/workload.yaml" which can't be opened by ReadFile